jep/200/README.adoc

```diff
@@ -276,7 +280,85 @@ For similar reasons, certain tests written in Groovy rather than Java prevent no
 
 == Reference Implementation
 
-link:https://github.com/jenkinsci/jenkins/pull/3120/files[Jenkins PR 3120] contains the bulk of the change and links to related PRs.
+* link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] contains the bulk of the change and links to related PRs.
+* link:https://github.com/jenkinsci/remoting/pull/208[Remoting PR #208] introduces the new API required to deliver the change.
+
+== Rollout plan
+
+=== Phase 1. Remoting release
+
+Target date: Dec 29, 2017
+
+JEP-200 requires extra API in Remoting, so link:https://github.com/jenkinsci/remoting/pull/208[pull request #208]
+needs to be delivered in advance.
+https://github.com/oleg-nenashev[@oleg-nenashev] will release the following components once the rollout plan is confirmed with the JEP sponsor:
+
+* Remoting
+* link:https://github.com/jenkinsci/docker-slave[docker-slave] (including the `latest` tag)
+* link:https://github.com/jenkinsci/docker-jnlp-slave[docker-jnlp-slave] images (including the `latest` tag)
+* link:https://github.com/jenkinsci/swarm-plugin/[Swarm Plugin Client]
+
+If everything goes fine, the Remoting release may be integrated into the weekly release
+(without actual use of the new API).
+
+=== Phase 2. Weekly Release
+
+Target date: Jan 13, 2017
+
+0. Once this JEP is approved, the https://jenkins.io/redirect/class-filter/ will be created on the Jenkins website
+** This document should provide a custom guide for creating JIRA issues with the `jep-200` label
+0. The JEP sponsor will write an announcement blogpost, which will describe the change and provide links to mitigation guidelines
+** This blog-post will be used as part of the upgrade guideline for LTS
+0. Once the blog post draft is approved,
+link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] will be integrated towards the next weekly release
+
+// oleg-nenashev: This process is a bit harsh for a single JEP sponsor, but in this case the sponsor represents a company AFAIK.
+// So I feel reasonable to set a maintenance quality bar taking a serious risk of regressions into account
+After the weekly release the JEP sponsor
+(or a group of people nominated by him, _JEP-200 maintainer(s)_)
+will be responsible to provide an extra support for the issues:
+
+* Issues with the `jep-200` label in Jenkins JIRA get first response within 2 business days (Monday-Friday)
```

Comment:
I do not think we should be setting SLAs. Obviously as soon as I become aware of any regression I would treat fixing it as a priority. No different from any other risky change to Jenkins.

Reply:
@daniel-beck WDYT? Do we need such SLA in terms of a JEP, which includes a risk of regressions? I am in favor of that, but having more votes would be useful

Reply:
We have no "SLA" today and somehow get notable regressions fixed anyway, so would not consider this necessary. This looks like an answer to a problem that doesn't exist.

Reply:
OK, I will rework it to exclude SLA in the current form